src/structures/Manager.ts

```diff
@@ -228,8 +228,49 @@ export class Manager extends EventEmitter {
       });
   }
 
-  /**
-   * Initiates the Manager class.
+    /**
+     * Returns the Node in same region as server.
+     * @param region
+     */
+    public nearestNode(region: string): Node {
+        const nodes = this.nodes
+            .filter((node) => {
+                if (!node.options.region)
+                    return node.connected
+                else if (Array.isArray(node.options.region))
+                    return node.connected && node.options.region.includes(region)
+                else
+                    return node.connected && node.options.region.toLowerCase() == region.toLowerCase()
+            });
+
+        if (!nodes)
+            return null;
+        else if (nodes.size === 1)
+            return nodes.first();
+        else if (nodes.size > 1)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585535)
```suggestion
        else if (nodes.size > 1)
```